src/plugins/vis_type_vega/public/vega_view/vega_base_view.js

```diff
@@ -230,14 +231,12 @@ export class VegaBaseView {
   }
 
   _addMessage(type, text) {
-    if (!this._$messages) {
-      this._$messages = $(`<ul class="vgaVis__messages">`).appendTo(this._$parentEl);
+    if (type === 'warn') {
+      this._notifications.toasts.addWarning(text);
+    }
+    if (type === 'err') {
+      this._notifications.toasts.addDanger(text);
```

Comment:
Do we need to destroy added toasts in the `_addDestroyHandler` (line 117) like before?